dask_searchcv/tests/test_adaptive.py

```diff
@@ -0,0 +1,179 @@
+from dask_searchcv.adaptive import _top_k, Hyperband
+from dask_ml.linear_model import PartialSGDClassifier
+from dask_ml.datasets import make_classification
+import numpy as np
+import dask.array as da
+from dask.distributed import Client
+import pandas as pd
+import pytest
+import distributed
+from pprint import pprint
+import scipy.stats as stats
+import random
+from sklearn.linear_model import Lasso
+
+
+class TestFunction:
+    def _fn(self):
+        return self.value
+
+    def get_params(self, deep=None, **kwargs):
+        return {k: getattr(self, k) for k, v in kwargs.items()}
+
+    def set_params(self, **kwargs):
+        for k, v in kwargs.items():
+            setattr(self, k, v)
+        return self
+
+    def partial_fit(self, *args, **kwargs):
+        pass
+
+    def score(self, *args, **kwargs):
+        return self._fn()
+
+
+def test_hyperband_sklearn():
+    client = _get_client()
+    X, y = make_classification(n_samples=1000, chunks=500)
+    classes = da.unique(y)
+    model = PartialSGDClassifier(warm_start=True, classes=classes,
```

Comment:
PartialSGDClassifier is being deprecated in favor of `Incremental(SGDClassifier())`.

Hopefully this will work for you (if not, let me know). It'll require passing through `fit_kwargs` like

```python
alg.fit(X, y, classes=[0, 1]
```

And you'll need to ensure that `classes` gets through to `Incremental.fit`, which passes it through to `SGDClassifier.partial_fit`.